### DISCUSSION TITLE
Controller

### DISCUSSION THREAD

**Discussion by mgstraface** [Sebastian-pz/ducker-api#74] [Q&A] [answered]
Tenemos que hacer un controller para manejar el tema de los reportes (tanto de publicaciones como de usuarios)

**Comment by Sebastian-pz** [comment]
Cómo lo tienes planteado, hacerlo por temas ejemplo: Contenido racista, zenofobo y así, o ¿Unicamente por número de reportes?

**Reply by MelHellrigl** [reply]
Opino que sea por temas como mencionó Seba.

**Comment by Sebastian-pz** [comment, marked as answer]
Okey, entonces el día de mañana haremos el sistema de reportes. Basado en el contenido de la cuenta y/o reporte. Feliz día.

**Comment by mgstraface** [comment]
entones queda así... mañana terminamos el reporte.
